packages/manager/modules/sms/src/sms/senders/add/telecom-sms-senders-add.controller.js

```diff
@@ -165,7 +167,15 @@ export default class {
           reason: this.sender.reason,
         },
       )
-      .$promise.then(() => this.$state.go('sms.service.senders'))
+      .$promise.then(() =>
+        this.sender.supportingDocument !== undefined
+          ? this.addSupportingDocument(
+              this.sender.sender,
+              this.sender.supportingDocument[0],
```

Comment:
don't you have to loop on every supporting document and add it ?

Reply:
No, just one supporting document is required for this feature.
(The component `oui-file` has an array to handle documents, whether it is for only one file or many).